src/Controller/Component/CsrfComponent.php

```diff
@@ -56,7 +56,7 @@ class CsrfComponent extends Component
         'cookieName' => 'csrfToken',
         'expiry' => 0,
         'secure' => false,
-        'httpOnly' => false,
+        'httpOnly' => true,
```

Comment:
Just in theory. Taken you submit a HTTP only via HTTPS only, how would XSS interact with that?